### ISSUE TITLE
Staging Review finding: File upload error not announced

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#123070]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Bio-Huntridgelabs
**Product:** VA Form 21P-8416
**Feature:** Form Digitization

## Findings details
**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** Yes
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

When selecting an unsupported file type in the file upload component, the error message is not announced.

<details><summary>Link, screenshot or steps to recreate</summary>

- Using NVDA with Firefox/Chrome
- Add any type of expense in the form flow
- On Submit your supporting documents, TAB to the file upload component
- Click Choose from folder
- Select an supported filetype such as a .csv file
- click "open" in the system window
- Note screen reader announcement
- Note: If the user clicks "Continue" then the error message is announced. 

<img width="828" height="508" alt="image upload file type error" src="https://github.com/user-attachments/assets/a16c6937-08cb-4487-b1ed-5a67aa9b2d35" />

</details>

## Recommended action

Ensure errors are announced, so that users of screen readers understand what needs to be fixed to proceed.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** [WCAG 3.3.1 Error Identification - Level A](https://www.w3.org/TR/WCAG22/#error-identification)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by RyanMunsch** [comment]
@caw310 this will spill over into next sprint due to requested changes in PR review and ongoing edge cases/inconsistencies with VoiceOver that are being caught and resolved.
